root/lib/systemd/system/titus-seccomp-agent@.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
+Description=Titus seccomp agent for container %i
+# If the service restarts more than 10 times in 30 seconds, let it die
+StartLimitIntervalSec=30
+StartLimitBurst=10
```

Comment:
Don't you want a conditionexists? as part of this?

Reply:
what would the condition be, that a path exists or something?